docs/pages/enroll-resources/application-access/okta/user-sync.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Okta user sync
+description: Details on Okta integration user synchronization
+---
+
+Teleport uses two complimentary methods for synchronizing Teleport resources with
+an upstream Okta Organization:
+
+- the Okta integration user sync process
+- provisioning via SCIM
+
+Okta user sync works in tandem with [SCIM integration](./scim-integration.mdx#how-it-works). Unlike
+SCIM integration, it is initiated from the Teleport side, and therefore requires API access to Okta
+and it is slower than SCIM, but it provides following benefits:
+
+- robustness:
+  - it does retries, it will fetch the users if for whatever reason SCIM request doesn't reach
+    the Teleport cluster
+  - it will fetch users that are assigned to the SSO app before SCIM was enabled
+- provides ability to synchronize users for the private Teleport clusters which are not accessible
+  over internet
+- unlocks enrollment of [synchronizing Okta apps and groups](./app-and-group-sync.mdx) as
+  Teleport Access Lists 
```

Comment:
I would provide a couple of quick reasons instead of a detailed, multi-level outline.